test/functional/test_framework/test_framework.py

```diff
@@ -929,6 +929,31 @@ def activate_dip0024(self, expected_activation_height=None):
         self.nodes[0].sporkupdate("SPORK_17_QUORUM_DKG_ENABLED", spork17_value)
         self.wait_for_sporks_same()
 
+    def activate_v19(self, expected_activation_height=None):
```

Comment:
please, check this one: https://github.com/knst/dash/commit/33b6d629f8d6b8b81f8c6f09036d57c4deeb5614

Reply:
Implemented your idea, but kept the sporks deactivation/reactivation only for dip0024